### PR TITLE
CU-en48kx_fewlines-eslint-config_Bump-eslint-config-prettier

### PR DESCRIPTION
**Pull request by nsallerin**
## Description

This PR bumps `eslint-config-prettier` dep. 

## Related Issue

[CU-en48kx](https://app.clickup.com/t/en48kx)

## Motivation and Context

We want to keep our package up to date.

## How Has This Been Tested?

Experimental version published and tested with Sparta-API.

## Types of changes

- Chore (non-breaking change which refactors / improves the existing code base)
- ~Bug fix (non-breaking change which fixes an issue)~
- ~New feature (non-breaking change which adds functionality)~
- ~Breaking change (fix or feature that would cause existing functionality to 
  change)~

## Checklist:

- ✅  My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- ✅  I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
- :red_circle: I have added tests to cover my changes.
- ✅  All new and existing tests passed.

[CONTRIBUTING_FILE]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-en48kx [@fewlines/eslint-config] Bump 'eslint-config-prettier'](https://app.clickup.com/t/en48kx)

**Comment by Fuzznimp** [comment]
Ok good for me, you can bump the package version and I'll approve it :).

Don't forget to update the CHANGELOG.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## 3.2.0 - 2021-02-22
```
Comment (Meyclem) [inline CHANGELOG.md:3]: @Fuzznimp what do you think about changing it to 4.0.0? It seems to be breaking in some cases.

Comment (nsallerin) [reply CHANGELOG.md:3]: and considering the fact that it works with an app which only bumps our package but keeps the previous `eslint-config-prettier`

Comment (Fuzznimp) [reply CHANGELOG.md:3]: Yeah I agree with you @Meyclem, it's a fix. Even if you override the eslint-config — let say to add alias patches — you still get the extended files (e.g. the `typescript.js `).

**Review by Meyclem — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Meyclem) [reply CHANGELOG.md:3]: 👍

**Review by Meyclem — APPROVED** [review]

**Pull request merged**